packages/notifications/src/elements/content/Close.tsx

```diff
@@ -20,7 +20,7 @@ export const Close = React.forwardRef<HTMLButtonElement, ButtonHTMLAttributes<HT
     const hue = useNotificationsContext();
 
     return (
-      <StyledClose ref={ref} hue={hue} aria-label={ariaLabel} {...props}>
+      <StyledClose ref={ref} hue={hue} aria-label={ariaLabel} {...props} aria-hidden>
```

Comment:
This is a controversial quality-of-life enhancement. With the assumption that most notifications are auto-dismissed and because we are against focus stealing, it's unlikely screen-reader users will be able to use the `Close` button. Therefore, I think it's better to remove it from the a11y tree to prevent its announcement.

Additionally, I could first check that the `autoDismiss` is `true` before applying `aria-hidden`. 

Thoughts? 🤔

Reply:
This doesn't feel right to me. Garden's approach to accessibility is a pragmatic one. If a close exists, there shouldn't be harm in announcing it. I'm also not aware of this being flagged during a11y audit – so best not to touch it.

Reply:
My only concern is your wording: "most notificaitons" - implying there are some notifications that may still receive focus, right?

Reply:
Sounds good. I'll revert.